devops/dependencies.json

```diff
@@ -19,9 +19,9 @@
       "root": "{DEPS_ROOT}/opencl/runtime/linux/oclgpu"
     },
     "level_zero": {
-      "github_tag": "v1.20.2",
-      "version": "v1.20.2",
-      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.20.2",
+      "github_tag": "v1.21.2",
```

Comment:
We are using driver 25.05.32567.17 which [depends](https://github.com/intel/compute-runtime/releases/tag/25.05.32567.17) on L0 version 1.20.2 (current version used here)

There is no ABI break/stuff that will screw up from using the new version, right?

Reply:
I believe so. @nrspruit can you confirm we can safely bump the loader?

Reply:
Yes, there is no ABI change, v1.21.2 only includes more functionality in the laoder and several bug fixes for the runtime. It is safe to update.